README.rst

```diff
@@ -1401,7 +1401,7 @@ Where ``$COMMIT_RANGE`` is your range of commits, eg ``$TRAVIS_COMMIT_RANGE`` or
 Exit codes
 ----------
 
-Exit code is build using following bit flags:
+Exit code is built using following bit flags:
```

Comment:
not sure, but isn't `build` the active form here, I'm not a native English speaker so I'm not sure

Reply:
Me ~~too~~ neither, so I thought I comment ;)

You are right @SpacePossum - `build` is the active from, but after `is` don't we suppose to use passive form?

Reply:
ah yes, looks like a good change to me :+1: